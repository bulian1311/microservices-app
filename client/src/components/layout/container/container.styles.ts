import styled from "styled-components";

export const Container = styled.div`
  display: flex;
  align-items: center;
  flex-direction: column;
  max-width: 65rem;
  margin: 0 auto;

  /* @media (max-width: 720px) {
    padding: 1em;
  } */
`;
