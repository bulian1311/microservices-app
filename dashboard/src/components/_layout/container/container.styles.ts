import styled from "styled-components";

export const Container = styled.div`
  display: flex;
  align-items: center;
  max-width: 88%;
  margin: 0 auto;
`;
