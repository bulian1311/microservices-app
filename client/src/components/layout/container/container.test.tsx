import * as React from "react";
import { render } from "@testing-library/react";
import { Container } from "./container.styles";

test("TextArea matces snapshot.", () => {
  const { baseElement } = render(<Container />);
  expect(baseElement).toMatchSnapshot();
});
