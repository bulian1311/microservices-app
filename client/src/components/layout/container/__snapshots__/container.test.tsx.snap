// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea matces snapshot. 1`] = `
<body>
  <div>
    <div
      class="sc-bdfBwQ hdUiEV"
    />
  </div>
</body>
`;
